crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/PassportedMapper.java

```diff
@@ -1,35 +1,50 @@
 package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
 
 import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.Passported;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.Applicant;
 import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.MaatApplicationExternal;
 
 import javax.validation.constraints.NotNull;
+import java.util.Objects;
 
 class PassportedMapper {
 
-    private static final boolean INCOME_SUPPORT_DEFAULT_FALSE = false;
-    private static final boolean STATE_PENSION_DEFAULT_FALSE = false;
-    private static final boolean JOB_SEEKER_DEFAULT_FALSE = false;
-    private static final boolean EMPLOYMENT_SUPPORT_DEFAULT_FALSE = false;
-    private static final boolean CLAIMED_BY_PARTNER_DEFAULT_FALSE = false;
+    private static final boolean FALSE = false;
+    private static final boolean TRUE = true;
     private static final Passported.WhoDwpChecked WHO_DWP_CHECKED_DEFAULT_NULL = null;
 
     @NotNull
     Passported map(MaatApplicationExternal crimeApplyMaatApplicationExternal) {
 
         Passported passported = new Passported();
 
-        passported.setBenefitIncomeSupport(INCOME_SUPPORT_DEFAULT_FALSE);
-        passported.setBenefitGuaranteedStatePension(STATE_PENSION_DEFAULT_FALSE);
-        passported.setJobSeeker(JOB_SEEKER_DEFAULT_FALSE);
-        passported.setBenefitEmploymentSupport(EMPLOYMENT_SUPPORT_DEFAULT_FALSE);
-        passported.setBenefitClaimedByPartner(CLAIMED_BY_PARTNER_DEFAULT_FALSE);
+        passported.setBenefitIncomeSupport(FALSE);
+        passported.setBenefitGuaranteedStatePension(FALSE);
+        passported.setBenefitJobSeeker(FALSE);
+        passported.setBenefitUniversalCredit(FALSE);
+        passported.setBenefitEmploymentSupport(FALSE);
+        passported.setBenefitClaimedByPartner(FALSE);
         passported.setWhoDwpChecked(WHO_DWP_CHECKED_DEFAULT_NULL);
 
-        if(!crimeApplyMaatApplicationExternal.getMeansPassport().isEmpty()){
+        if (!crimeApplyMaatApplicationExternal.getMeansPassport().isEmpty()) {
             passported.setMeansPassport(String.valueOf(crimeApplyMaatApplicationExternal.getMeansPassport().get(0)));
```

Comment:
I know this isn't directly related to one of your changes, the getMeansPassport is a list which could technically include multiple elements. Assuming we should only ever receive a list with 1 element (arguably a design issue on Crime Apply Datastore side) I think  we should add validation for the same so as not to propagate any data quality issues.
i.e. if getMeansPassport().size() > 1; throw IllegalStateException("Invalid MeansPassport value")

Reply:
Yes, we will receive 1 element or empty for Means Passport from CA datastore. I think the current validation is OK.